### PR TITLE
fix(refs DPLAN 12667) move receiver address into globalconfig - env variable

### PR DESCRIPTION
**Pull request by MoritzMandler**
Ticket:
https://demoseurope.youtrack.cloud/issue/DPLAN-12667

Description:
move receiver address into globalconfig - env variable

- [ ] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/EventSubscriber/ProcedureMetricsEmailEventSubscriber.php [hunk 1]
```diff
@@ -38,7 +37,7 @@ public function __construct(
         private readonly CustomerService $customerService,
         private readonly CurrentUserService $currentUser,
         private readonly MailService $mailService,
-        private readonly GlobalConfigInterface $globalConfig,
+        private readonly GlobalConfig $globalConfig,
```
Comment (mussbach) [inline demosplan/DemosPlanCoreBundle/EventSubscriber/ProcedureMetricsEmailEventSubscriber.php:40]: I would prefer to keep the Interface and the temporary warning as long as the method is not yet added to the interface. Or what was the reasoning behind this?

**Review by mussbach — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/EventSubscriber/ProcedureMetricsEmailEventSubscriber.php (hunk 1 above)
Comment (MoritzMandler) [reply demosplan/DemosPlanCoreBundle/EventSubscriber/ProcedureMetricsEmailEventSubscriber.php]: That was the reason - changed in 43b6bf7bde25580984d2a42de5a4e2c84463c618

Comment (MoritzMandler) [reply demosplan/DemosPlanCoreBundle/EventSubscriber/ProcedureMetricsEmailEventSubscriber.php]: also changed in planed pick in 176c41f5a52a94ab161f12fb7e17a00371674f78

**Review by mussbach — APPROVED** [review]
lgtm thx

**Pull request merged**